src/util/meatballs.js

```diff
@@ -42,10 +44,38 @@ export const unregisterMeatballItem = id => {
   $(`[data-xkit-meatball-button="${id}"]`).remove();
 };
 
+/**
+ * Add a custom button to blogs' meatball menus in blog cards and the blog view header.
+ *
+ * @param {object} options - Destructured
+ * @param {string} options.id - Identifier for this button (must be unique)
+ * @param {string|Function} options.label - Button text to display. May be a function accepting the blog data of the post element being actioned on.
+ * @param {Function} options.onclick - Button click listener function
+ * @param {Function} [options.postFilter] - Filter function, called with the blog data of the menu element being actioned on. Must return true for button to be added. Some blog data fields, such as "followed," are not available in blog cards.
+ */
+export const registerBlogMeatballItem = function ({ id, label, onclick, postFilter }) {
+  blogMeatballItems[id] = { label, onclick, postFilter };
```

Comment:
1. I've regretted the casing of `onclick` for a while now
2. `postFilter` doesn't make sense if we're not filtering posts :P

```suggestion
 * @param {Function} options.onClick - Button click listener function
 * @param {Function} [options.blogFilter] - Filter function, called with the blog data of the menu element being actioned on. Must return true for button to be added. Some blog data fields, such as "followed", are not available in blog cards.
 */
export const registerBlogMeatballItem = function ({ id, label, onClick, blogFilter }) {
  blogMeatballItems[id] = { label, onClick, blogFilter };
```

(I thought about suggesting `postFilter` &rarr; `filter` but I probably had a reason for not going for `filter` for post meatball items, maybe to avoid confusion with `Array.prototype.filter`?)

Reply:
I'll merge then take a look!

Reply:
Ah, I see; changing to `onClick` in the current meatballs utils would require a big diff.